go-1.22.yaml

```diff
@@ -92,3 +92,18 @@ update:
     strip-prefix: go
     tag-filter: go1.22
     use-tag: true
+
+test:
+  environment:
+    contents:
+      packages:
+        - busybox
+  pipeline:
+    - name: Verify Go version
+      runs: |
+        go version | grep "go1.22"
```

Comment:
```suggestion
        go version | grep "go${{package.version}}"
```

Reply:
Most of these are on auto version streams these will cause breakages in the test when we move to 1.23 automatically